resources/styles/book-content/fun-in-physics.less

```diff
@@ -0,0 +1,9 @@
+.tutor-book-content {
+  .fun-in-physics {
+    margin-top: 20px;
+    padding: 0 @tutor-card-body-padding-horizontal;
+    & > .title:first-child {
+      .tutor-book-content.tutor-reading-main-title();
```

Comment:
The `.tutor-book-content` prefix was needed so that the variables and mixins defined inside the `.tutor-book-content` could be accessed by a file that was re-opening the mixin vs being included from inside it.  I've now decided that we don't need to be that extreme and am moving the includes back inside `tutor-book-content`